src/frontend/komponenter/Felleskomponenter/PdfVisningModal/useForhåndsvisning.ts

```diff
@@ -0,0 +1,62 @@
+import { IBrevData } from '../Hendelsesoversikt/BrevModul/typer';
+import {
+    byggDataRessurs,
+    byggFeiletRessurs,
+    byggHenterRessurs,
+    byggTomRessurs,
+    Ressurs,
+    RessursStatus,
+} from '@navikt/familie-typer';
+import { AxiosError } from 'axios';
+import React, { useEffect, useState } from 'react';
+import { FamilieAxiosRequestConfig, useApp } from '../../../context/AppContext';
+
+const useForhåndsvisning = () => {
+    const { axiosRequest } = useApp();
+
+    const [visForhåndsvisningModal, settVisForhåndsviningModal] = useState<boolean>(false);
+
+    const [hentetForhåndsvisning, settHentetForhåndsvisning] = React.useState<Ressurs<string>>(
+        byggTomRessurs()
+    );
+
+    useEffect(() => {
+        if (hentetForhåndsvisning.status === RessursStatus.SUKSESS) {
+            settVisForhåndsviningModal(true);
```

Comment:
Gir det kanskje litt mer mening om denne ligger inne i api-requesten sin then() clause?

Reply:
Ja, vi var litt frem og tilbake på denne, og kom frem til at vi fikk se hva du sa 😄  men var vel for å være helt sikker på at den ikke viser en modal uten data (før staten er satt), men tviler på at det oppstår noe lag her at det skjer.

Reply:
Den er flyttet inn i then clausen.